src/Agent/NewRelic/Profiler/Configuration/Configuration.h

```diff
@@ -171,15 +171,15 @@ namespace NewRelic { namespace Profiler { namespace Configuration {
                 });
         }
 
-        bool ShouldInstrument(xstring_t const& processPath, xstring_t const& appPoolId, xstring_t const& commandLine, bool isCoreClr)
+        bool ShouldInstrument(xstring_t const& processPath, xstring_t const& parentProcessPath, xstring_t const& appPoolId, xstring_t const& commandLine, bool isCoreClr)
         {
             if (isCoreClr)
             {
-                return ShouldInstrumentNetCore(processPath, appPoolId, commandLine);
+                return ShouldInstrumentNetCore(processPath, parentProcessPath, appPoolId, commandLine);
             }
             else
             {
-                return ShouldInstrumentNetFramework(processPath, appPoolId);
+                return ShouldInstrumentNetFramework(processPath, parentProcessPath, appPoolId);
```

Comment:
I don't believe .NET Framework applications can be hosted out of process in IIS, but I can't see this hurting anything...

Reply:
I'll take the `parentProcessPath` out of the function signature and just pass an empty string here to make it more obvious.